<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>GaussSeidel</class>
 <widget class="QMainWindow" name="GaussSeidel">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>756</width>
    <height>327</height>
   </rect>
  </property>
  <property name="palette">
   <palette>
    <active>
     <colorrole role="Base">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>255</red>
        <green>255</green>
        <blue>127</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="Window">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>170</red>
        <green>0</green>
        <blue>0</blue>
       </color>
      </brush>
     </colorrole>
    </active>
    <inactive>
     <colorrole role="Base">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>255</red>
        <green>255</green>
        <blue>127</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="Window">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>170</red>
        <green>0</green>
        <blue>0</blue>
       </color>
      </brush>
     </colorrole>
    </inactive>
    <disabled>
     <colorrole role="Base">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>170</red>
        <green>0</green>
        <blue>0</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="Window">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>170</red>
        <green>0</green>
        <blue>0</blue>
       </color>
      </brush>
     </colorrole>
    </disabled>
   </palette>
  </property>
  <property name="font">
   <font>
    <family>MV Boli</family>
   </font>
  </property>
  <property name="windowTitle">
   <string>GaussSeidel</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>450</x>
      <y>60</y>
      <width>21</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>22</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>=</string>
    </property>
   </widget>
   <widget class="QWidget" name="layoutWidget_5">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>100</y>
      <width>99</width>
      <height>24</height>
     </rect>
    </property>
    <layout class="QFormLayout" name="formLayout_24">
     <item row="0" column="1">
      <widget class="QDoubleSpinBox" name="E22_2">
       <property name="minimum">
        <double>-99.989999999999995</double>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="layoutWidget_6">
    <property name="geometry">
     <rect>
      <x>490</x>
      <y>150</y>
      <width>83</width>
      <height>24</height>
     </rect>
    </property>
    <layout class="QFormLayout" name="formLayout_25">
     <item row="0" column="0">
      <widget class="QLabel" name="label_27">
       <property name="font">
        <font>
         <pointsize>10</pointsize>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="text">
        <string>D</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QDoubleSpinBox" name="R4_2">
       <property name="minimum">
        <double>-99.989999999999995</double>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QPushButton" name="procesar">
    <property name="geometry">
     <rect>
      <x>580</x>
      <y>210</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Procesar</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>266</x>
      <y>210</y>
      <width>161</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>NÃšMERO DE ITERACIONES</string>
    </property>
   </widget>
   <widget class="QWidget" name="layoutWidget_15">
    <property name="geometry">
     <rect>
      <x>340</x>
      <y>60</y>
      <width>100</width>
      <height>24</height>
     </rect>
    </property>
    <layout class="QFormLayout" name="formLayout_33">
     <item row="0" column="1">
      <widget class="QDoubleSpinBox" name="E13_2">
       <property name="minimum">
        <double>-99.989999999999995</double>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QPushButton" name="salir">
    <property name="geometry">
     <rect>
      <x>660</x>
      <y>230</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Salir</string>
    </property>
   </widget>
   <widget class="QWidget" name="layoutWidget_10">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>60</y>
      <width>99</width>
      <height>24</height>
     </rect>
    </property>
    <layout class="QFormLayout" name="formLayout_28">
     <item row="0" column="1">
      <widget class="QDoubleSpinBox" name="E12_2">
       <property name="minimum">
        <double>-99.989999999999995</double>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="layoutWidget_7">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>150</y>
      <width>100</width>
      <height>24</height>
     </rect>
    </property>
    <layout class="QFormLayout" name="formLayout_26">
     <item row="0" column="1">
      <widget class="QDoubleSpinBox" name="E30_2">
       <property name="minimum">
        <double>-99.989999999999995</double>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="layoutWidget_9">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>150</y>
      <width>100</width>
      <height>24</height>
     </rect>
    </property>
    <layout class="QFormLayout" name="formLayout_27">
     <item row="0" column="1">
      <widget class="QDoubleSpinBox" name="E31_2">
       <property name="minimum">
        <double>-99.989999999999995</double>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="layoutWidget_18">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>60</y>
      <width>100</width>
      <height>24</height>
     </rect>
    </property>
    <layout class="QFormLayout" name="formLayout_36">
     <item row="0" column="1">
      <widget class="QDoubleSpinBox" name="E10_2">
       <property name="minimum">
        <double>-99.989999999999995</double>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QPushButton" name="borrar">
    <property name="geometry">
     <rect>
      <x>660</x>
      <y>200</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Borrar</string>
    </property>
   </widget>
   <widget class="QWidget" name="layoutWidget_13">
    <property name="geometry">
     <rect>
      <x>340</x>
      <y>10</y>
      <width>100</width>
      <height>24</height>
     </rect>
    </property>
    <layout class="QFormLayout" name="formLayout_31">
     <item row="0" column="1">
      <widget class="QDoubleSpinBox" name="E03_2">
       <property name="minimum">
        <double>-99.989999999999995</double>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="layoutWidget_4">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>150</y>
      <width>99</width>
      <height>24</height>
     </rect>
    </property>
    <layout class="QFormLayout" name="formLayout_23">
     <item row="0" column="1">
      <widget class="QDoubleSpinBox" name="E32_2">
       <property name="minimum">
        <double>-99.989999999999995</double>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QTextEdit" name="Texto">
    <property name="geometry">
     <rect>
      <x>600</x>
      <y>20</y>
      <width>111</width>
      <height>141</height>
     </rect>
    </property>
   </widget>
   <widget class="QWidget" name="layoutWidget_12">
    <property name="geometry">
     <rect>
      <x>340</x>
      <y>150</y>
      <width>100</width>
      <height>24</height>
     </rect>
    </property>
    <layout class="QFormLayout" name="formLayout_30">
     <item row="0" column="1">
      <widget class="QDoubleSpinBox" name="E33_2">
       <property name="minimum">
        <double>-99.989999999999995</double>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="layoutWidget_8">
    <property name="geometry">
     <rect>
      <x>340</x>
      <y>100</y>
      <width>100</width>
      <height>24</height>
     </rect>
    </property>
    <layout class="QGridLayout" name="gridLayout_2">
     <item row="0" column="0">
      <widget class="QDoubleSpinBox" name="E23_2">
       <property name="minimum">
        <double>-99.989999999999995</double>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QDoubleSpinBox" name="err">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>230</y>
      <width>62</width>
      <height>22</height>
     </rect>
    </property>
   </widget>
   <widget class="QWidget" name="layoutWidget_16">
    <property name="geometry">
     <rect>
      <x>490</x>
      <y>10</y>
      <width>82</width>
      <height>24</height>
     </rect>
    </property>
    <layout class="QFormLayout" name="formLayout_34">
     <item row="0" column="0">
      <widget class="QLabel" name="label_37">
       <property name="font">
        <font>
         <pointsize>10</pointsize>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="text">
        <string>A</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QDoubleSpinBox" name="R1_2">
       <property name="minimum">
        <double>-99.989999999999995</double>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="layoutWidget_14">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>10</y>
      <width>99</width>
      <height>24</height>
     </rect>
    </property>
    <layout class="QFormLayout" name="formLayout_32">
     <item row="0" column="1">
      <widget class="QDoubleSpinBox" name="E02_2">
       <property name="minimum">
        <double>-99.989999999999995</double>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="layoutWidget_11">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>100</y>
      <width>100</width>
      <height>24</height>
     </rect>
    </property>
    <layout class="QFormLayout" name="formLayout_29">
     <item row="0" column="1">
      <widget class="QDoubleSpinBox" name="E21_2">
       <property name="minimum">
        <double>-99.989999999999995</double>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>490</x>
      <y>100</y>
      <width>82</width>
      <height>24</height>
     </rect>
    </property>
    <layout class="QFormLayout" name="formLayout_20">
     <item row="0" column="0">
      <widget class="QLabel" name="label_22">
       <property name="font">
        <font>
         <pointsize>10</pointsize>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="text">
        <string>C</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QDoubleSpinBox" name="R3_2">
       <property name="minimum">
        <double>-99.989999999999995</double>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="layoutWidget_20">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>100</y>
      <width>100</width>
      <height>24</height>
     </rect>
    </property>
    <layout class="QFormLayout" name="formLayout_38">
     <item row="0" column="1">
      <widget class="QDoubleSpinBox" name="E20_2">
       <property name="minimum">
        <double>-99.989999999999995</double>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>210</y>
      <width>151</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>PORCENTAJE DE ERROR</string>
    </property>
   </widget>
   <widget class="QWidget" name="layoutWidget_19">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>60</y>
      <width>100</width>
      <height>24</height>
     </rect>
    </property>
    <layout class="QFormLayout" name="formLayout_37">
     <item row="0" column="1">
      <widget class="QDoubleSpinBox" name="E11_2">
       <property name="minimum">
        <double>-99.989999999999995</double>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QDoubleSpinBox" name="x">
    <property name="geometry">
     <rect>
      <x>300</x>
      <y>230</y>
      <width>62</width>
      <height>22</height>
     </rect>
    </property>
   </widget>
   <widget class="QWidget" name="layoutWidget_3">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>10</y>
      <width>100</width>
      <height>24</height>
     </rect>
    </property>
    <layout class="QFormLayout" name="formLayout_22">
     <item row="0" column="1">
      <widget class="QDoubleSpinBox" name="E01_2">
       <property name="minimum">
        <double>-99.989999999999995</double>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="layoutWidget_17">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>100</width>
      <height>24</height>
     </rect>
    </property>
    <layout class="QFormLayout" name="formLayout_35">
     <item row="0" column="1">
      <widget class="QDoubleSpinBox" name="E00_2">
       <property name="minimum">
        <double>-99.989999999999995</double>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="layoutWidget_2">
    <property name="geometry">
     <rect>
      <x>490</x>
      <y>60</y>
      <width>82</width>
      <height>24</height>
     </rect>
    </property>
    <layout class="QFormLayout" name="formLayout_21">
     <item row="0" column="0">
      <widget class="QLabel" name="label_23">
       <property name="font">
        <font>
         <pointsize>10</pointsize>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="text">
        <string>B</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QDoubleSpinBox" name="R2_2">
       <property name="minimum">
        <double>-99.989999999999995</double>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>756</width>
     <height>23</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuGauss_Seidel">
    <property name="title">
     <string>Gauss-Seidel</string>
    </property>
   </widget>
   <addaction name="menuGauss_Seidel"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
